components/shared/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# build, lint, and test
+FROM node:12.4 as builder
+WORKDIR /usr/app/builder
+COPY ./components/stream-provider ./components/stream-provider
+COPY ./components/shared ./components/shared/
+COPY ./package*.json ./
+COPY ./lerna.json ./
+RUN npm install lerna -g
+RUN echo y | lerna clean
```

Comment:
nit: using [`yes`](https://en.wikipedia.org/wiki/Yes_%28Unix%29) is probably a more common pattern here

Reply:
lerna clean only accepts a y or n on prompt so we are stuck with "y"